index.js

```diff
@@ -23,9 +25,20 @@ const hbsPartialFile = handlebarsPartialFile({
 
 hbsPartialFile.registerDirectory('badge', 'svg');
 
-// eslint-disable-next-line quotes
-const CSP_HEADER = "default-src 'none'; base-uri 'none'; form-action 'self'; frame-src 'self'; frame-ancestors 'self'; img-src 'self'; manifest-src 'self'; style-src 'self'";
-const REFERRER_HEADER = 'no-referrer, strict-origin-when-cross-origin';
+const CSP = {
```

Comment:
I wonder how blankie deals with object property ordering, given this here is alphabetically sorted (and that they don't have an order in JavaScript in the first place).

I think it's somewhat important the default-src is sent first. Does that happen with this patch? :thinking:

Reply:
Are you 100% sure the order matters? This patch doesn't send `default-src` first, but I could change the order. As long as it does matter

Reply:
The order doesn't matter for browser parsing, actually. Just for reading it :-) Nevermind..